src/verifier/mem_encode/encoder.hpp

```diff
@@ -264,15 +274,33 @@ class encoder {
       write_mol(atom, LMN_ATTR_MAKE_LINK(0), -1, new_bsptr, visited, TRUE);
       if (new_bsptr.is_valid()) {
         /* atomからたどった分子が書き込みに成功したので次のアトムの書き込みを試みる */
```

Comment:
ここのコメントも以前のものに差し戻しておいてください
```suggestion
        /* atomからたどった分子が書き込みに成功したので、last_validに記憶する */
```